include/lsst/afw/table/IdFactory.h

```diff
@@ -48,6 +48,17 @@ class IdFactory {
      */
     static std::shared_ptr<IdFactory> makeSource(RecordId expId, int reserved);
 
+    /**
+     * Return the number to pass as the 'reserved' argument to makeSource for
+     * an exposure ID with the given maximum number of bits.
+     *
+     * @param[in]  maxBits  The maximum number of bits an exposure ID can have.
```

Comment:
"exposure ID" does confuse me at times since I sometimes see this used ambiguously.  Is this "detector exposure ID" or "exposure ID"? The camera mappers define nbits for detector exposure ID and not exposure ID.  Does the system complain if mapper nbits exceeds this maxBits?

Reply:
This is sometimes detector_exposure_id and sometimes it has nothing at all to do with exposures (except maybe `afw.image.Exposure` of a coadd).

Agree that it's confusing, but it's a much bigger issue than this ticket.

Reply:
Oh, and yes, you will get an exception if the numbers of bits don't add up.